crates/winscard/Cargo.toml

```
@@ -32,6 +32,8 @@ rand_core = "0.6.4"
 sha1 = { version = "0.10.6", default-features = false }
 base64 = { version = "0.21" , optional = true }
 picky-asn1-der = { version = "0.4", optional = true }
+num-derive = "0.4.2"
+num-traits = { version = "0.2.18", default-features = false }
```

nitpick: Only specify the minor number.
```suggestion
num-derive = "0.4"
num-traits = { version = "0.2", default-features = false }
```

---

done